README.md

```diff
@@ -20,15 +25,15 @@ Bandwidth: 1.2 GB/s RX | 990 MB/s TX
 Bandwidth: 1.2 GB/s RX | 944 MB/s TX
 ```
 
-on all the servers IP1 IP2 IP3 ... respectively.
-
-Default ports are `9999` and `10000` make sure your firewalls allow these ports. You may optionally configure `./hperf` to use custom ports as well, for example setting port `5001` would require opening up port `5002` as well.
-
+Default ports are `9999` make sure your firewalls allow this port. You may optionally configure `./hperf` to use a custom port as well `-port MYPORT`
 
 ```
-NPERF_PORT=5001 ./hperf IP1 IP2 IP3 ...
+./hperf -port MYPORT IP1 IP2 IP3 ...
 ```
 
+Two preconfigured latency tests have been added `-longLatencyTest` and `-shortLatencyTest`.<br>
```

Comment:
```suggestion
```

No need to say it like a changelog, this is just a doc to indicate what is already there.